core/src/com/unciv/ui/tilegroups/TileGroup.kt

```diff
@@ -587,14 +587,20 @@ open class TileGroup(
 
     private fun updateTileColor(isViewable: Boolean) {
         val baseTerrainColor = when {
+            isViewable && tileInfo.isPillaged() && tileSetStrings.tileSetConfig.useColorAsBaseTerrain -> tileInfo.getBaseTerrain().getColor().lerp(Color.BROWN, 0.6f)
+            isViewable && tileInfo.isPillaged() -> Color.WHITE.cpy().lerp(Color.BROWN, 0.6f)
```

Comment:
??? Why do you change the entire base terrain color for pillaged improvements? Surely only the improvement icon should be changed?

Reply:
To make it obvious that the tile has a pillaged improvement/road on it. If the Road is pillaged, coloration is too small. And depending on the improvement sprite could be too indistinct as well. Coloring the whole tile makes it more obvious, similar to Civ5 the entire tile becomes like a wasteland